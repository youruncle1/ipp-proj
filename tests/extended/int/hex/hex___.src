.IPPcode23
pushs int@0X1f_A53cA