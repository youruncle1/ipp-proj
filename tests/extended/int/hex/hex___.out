<?xml version="1.0" encoding="UTF-8"?>
<program language="IPPcode23">
  <instruction order="1" opcode="PUSHS">
    <arg1 type="int">0X1f_A53cA</arg1>
  </instruction>
</program>